backend/src/main/java/pl/edu/agh/apdvbackend/auth/SecurityConfig.java

```diff
@@ -27,6 +33,20 @@ protected void configure(AuthenticationManagerBuilder auth)
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        super.configure(http);
+        http.csrf().disable();
```

Comment:
nie bedziemy miec klientów przeglądarkowych?

Reply:
To `csrf` dałem na `disable`, bo głównie podążałem za tutorialem, a u nas JWT pełni rolę tokena, dzięki któremu dostajemy dostęp do zasobów

Reply:
oki, doczytalam że to się wyłącza jak nie ma browser clients, więc można nad tym jeszcze pomyśleć, co prawda nie wiem co flutter na to, ale wszystko da sie zrobić